src/app/clusters/door-lock-server/door-lock-server-user.c

```diff
@@ -265,10 +267,10 @@ bool emberAfDoorLockClusterSetPinCallback(uint16_t userId, uint8_t userStatus, u
                                (uint8_t *) &rfProgrammingEventMask, sizeof(rfProgrammingEventMask));
     if ((rfProgrammingEventMask & BIT(2)) && !status && (pin != NULL))
     {
-        emberAfFillCommandDoorLockClusterProgrammingEventNotification(EMBER_ZCL_DOOR_LOCK_EVENT_SOURCE_RF,
-                                                                      EMBER_ZCL_DOOR_LOCK_PROGRAMMING_EVENT_CODE_PIN_ADDED, userId,
-                                                                      pin, userType, userStatus, emberAfGetCurrentTime(), pin);
-        SEND_COMMAND_UNICAST_TO_BINDINGS();
+        emberAfFillCommandDoorLockClusterProgrammingEventNotification(
+            EMBER_ZCL_DOOR_LOCK_EVENT_SOURCE_RF, EMBER_ZCL_DOOR_LOCK_PROGRAMMING_EVENT_CODE_PIN_ADDED, userId, pin, userType,
+            userStatus, 0, pin); /*emberAfGetCurrentTime() #2507*/
```

Comment:
The comment should be right after the `0`; otherwise it's not clear what it refers to.